backend/app/routers/user.py

```diff
@@ -1,32 +1,72 @@
 from typing import List
 
-from fastapi import Depends, HTTPException, APIRouter
+from fastapi import Depends, HTTPException, APIRouter, status
+from fastapi.responses import JSONResponse, Response
 from sqlalchemy.orm import Session
 
-from .. import crud, schemas
+from .. import auth, crud, models, schemas
 from ..database import get_db
 
 
 router = APIRouter()
 
 
-@router.get("/users/", response_model=List[int])
+@router.get("/", response_model=List[int])
 def get_users(skip: int = 0, limit: int = 100, db: Session = Depends(get_db)):
+    """Endpoint to get a list of user ids."""
     return [user.user_id for user in crud.get_users(db, skip=skip, limit=limit)]
 
 
-@router.get("/users/{user_id}", response_model=schemas.User)
+@router.get("/{user_id}", response_model=schemas.User)
```

Comment:
Should this path start with "/users"?

Reply:
Ignore, didn't see the prefixes part before commenting :)

Reply:
Everything inside routers/user.py is prefixed with "/users" see 
`app.include_router(user.router, prefix="/users")` in main.py